app/models/user.rb

```diff
@@ -2,12 +2,12 @@
 
 class User < ApplicationRecord
   USER_ROLES = {
-    user: 'user',
-    admin: 'admin',
-    super_admin: 'super_admin'
+    user: 0,
+    admin: 1,
+    super_admin: 2
```

Comment:
Have seen these changes within another PR: https://github.com/mmrshk/innovation_forpost/pull/47/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29b

Reply:
the fact is that the previous version did not work, which is why it was not possible to do a design